modules/ocl/cl_geo_map_handler.h

```diff
@@ -135,6 +136,13 @@ class CLGeoMapHandler
         return PointFloat2 (0.0f, 0.0f);
     }
 
+    virtual const BowlDataConfig &get_bowl_config () {
+        XCAM_ASSERT (false && "CLGeoMapHandler::get_bowl_config is not supported");
+        BowlDataConfig bowl_data_config;
```

Comment:
This function should be rejected.
BTW, this looks like hardcode and usage of variable reference is wrong. C++reference(&) is similar like pointer.  bowl_data_config is a local variable, after function returned, the reference is damaged. the return value can be any values. BE AWARE OF THAT. if your result is not correct, it's much possible caused by this function.